website/docs/sigs.md

```diff
@@ -250,6 +250,46 @@ end
 Main.name_length(Main.greet('Alice')) # => error!
 ```
 
+## Class Methods
+
+There are at least three ways to define class (static) methods in ruby.
+The placement of the `extend T::Sig` varies accordingly.
+
+1. `def self.greet`
+
+```ruby
+class Main
+  extend T::Sig
+  
+  sig {params(name: String).void}
+  def self.greet(name)
+    puts "Hello, #{name}!"
+  end
+end
+```
+
+2. `class << self`
+
+```ruby
+class Main
+  class << self
+    extend T::Sig
+    sig {params(name: String).void}
+    def greet(name)
+      # etc.
+```
+
+3. `def Main.greet`
+
+```ruby
+extend T::Sig
+sig {params(name: String).void}
+def main.greet(name)
+  # etc.
+```
+
+The third style, `def Main.greet`, should be avoided, because it pollutes the global namespace.
```

Comment:
On line 282 and 291, you use `Main.greet`, but in the code on line 287 you use `main.greet`. Is that significant?

Reply:
Nah, I meant to use capital `Main`